CMSgov/mdct-mcr
# MDCT-1962: Modify ChoiceList choices to store name and value

## Description
Fixes [MDCT-1962](https://qmacbis.atlassian.net/browse/MDCT-1962) Choices have even more meaning

https://user-images.githubusercontent.com/19439679/191330006-5158ad19-6f0f-4fe0-9da4-a99165eeda54.mov

### How to test
./dev local
Create a program
Go to a page with radio buttons or checkboxs
Click and enter info to your hearts content
Save the form
Come back and make sure its hydrated!

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
